### DISCUSSION TITLE
Flows: log or Read data before delete?

### DISCUSSION THREAD

**Discussion by dkamali-mc** [directus/directus#14856] [Archive]
I have a flow that monitors collection events (create, update, delete). I would like to read or log the item before deleting it.

Is that possible? only knowing the ID of the deleted item is not sufficient 

![image](https://user-images.githubusercontent.com/75494069/182940302-485b46f4-5ce8-4b0d-bbb0-c91bd4cad452.png)

**Comment by rijkvanzanten** [comment]
> Is that possible? only knowing the ID of the deleted item is not sufficient

If you're using a `filter` event as the trigger, you can read the item from the database using the `read` operation before it's deleted 👍🏻
